setup.py

```diff
@@ -64,26 +65,45 @@ def finalize_options(self):
         # Get the value of the skip-driver parameter from the install command.
         self.set_undefined_options("install", ("skip_driver", "skip_driver"))
 
+    def _build_linux_driver(self):
+        log.info("building the linux driver")
+        build_driver = os.path.join(self.real_build_lib, "drivers", "linux")
+        ko_ext = os.path.join(build_driver, "chipsec.ko")
+        # We copy the drivers extension to the build directory.
+        self.copy_tree(os.path.join("drivers", "linux"), build_driver)
+        # Run the makefile there.
+        subprocess.check_output(["make", "-C", build_driver])
+        # And copy the resulting .ko to the right place.
+        # That is to the source directory if we are in "develop" mode,
+        # otherwise to the helper subdirectory in the build directory.
+        root_dst = "" if self.inplace else self.real_build_lib
+        dst = os.path.join(root_dst, "chipsec", "helper", "linux")
+        self.copy_file(ko_ext, dst)
+        # Finally, we clean up the build directory.
+        shutil.rmtree(os.path.join(self.real_build_lib, "drivers"))
+
+    def _build_darwin_driver(self):
+        log.info("building the OSX driver")
+        build_driver = os.path.join(self.real_build_lib, "drivers", "osx")
+        xcodeproject = os.path.join(build_driver, "chipsec.xcodeproj")
+        self.copy_tree(os.path.join("drivers", "osx"), build_driver)
+        subprocess.check_output(["xcodebuild", "-project", xcodeproject,
```

Comment:
Same here. `check_output` will verify the value of the return code and raise CalledProcessError if it failed.